ifacemaker.go

```diff
@@ -5,45 +5,68 @@ import (
 	"io/ioutil"
 	"log"
 
+	"github.com/kazu/ifacemaker/maker"
```

Comment:
It seems like you refer your own repo here. for pull request it should be reverted back to original.

Reply:
ok . fix later